src/scss/ckeditor.scss

```diff
@@ -9,11 +9,174 @@
 @import '01_variables/_index';
 @import '02_mixins/_index';
 
-// /* Content is created after this point */
-@import '03_generic/_index';
-@import '04_elements/_index';
-@import '05_objects/_index';
-@import '06_components/paragraphs/text';
+.ck-content {
+  @import '03_generic/_index';
+
+  // Text paragraph styles as written in 03_generic/_base.scss and '06_components/paragraphs/text'.
+  @include font('body');
+  color: $color-black-90;
+  font-family: $base-font-family;
+
+  > div {
+    max-width: $content-width-max;
+  }
+
+  @import '04_elements/_index';
+
+  // Import only the styles that are needed for the ckeditor content creation and not the whole __index.scss
+  @import '05_objects/_quote';
+}
+
+// Additional styling for links.
+// The CKEditor link element dom doesn't have all the elements that the rendered version has so we are adding the
+// icons here for the links so that the links look similar as in the end result.
+.ck-content a {
+  text-decoration: underline;
+
+  // External link styles
+  &[href^='https://']::after {
```

Comment:
Also, having `https://` in front of a link is not enough to detect it as an external link.  This approach would mark links with https://www.hel.fi/fi as external.

We should run the actual script that checks if the link is external or not. Or if we really need to do it with css, we should check for our common urls like:
```
&:is([href^='http://'],[href^='https://']):not([href^='http://hel.fi/'],[href^='https://hel.fi/'],[href^='http://www.hel.fi/'],[href^='https://www.hel.fi/'])::after {
```
But I would recommend against it as then external teams using our HDBT code would be tied to these hardcoded values.

Reply:
Yeah this is tricky - maybe we can add a script to run with the ckeditor and modify its dom but it would get this issue quite heavily out of estimate so I think its best if we just leave the external link styles away. This is an improvement already as it is compared to the current ckeditor interface.